indexing-service/src/main/java/org/apache/druid/indexing/worker/WorkerTaskManager.java

```diff
@@ -179,6 +179,16 @@ public Map<String, TaskAnnouncement> getCompletedTasks()
     return completedTasks;
   }
 
+  public Map<String, TaskDetails> getRunningTasks()
+  {
+    return runningTasks;
```

Comment:
No need to expose the `TaskDetails` class as no other field of the `TaskDetails` class is being used in the monitoring. Might as well just do the following :

```suggestion
    return CollectionUtils.mapValues(runningTasks, detail -> detail.task);
```